src/test/scala/nl.knaw.dans.easy.bag2deposit/AppSpec.scala

```diff
@@ -17,29 +17,32 @@ package nl.knaw.dans.easy.bag2deposit
 
 import better.files.File
 import nl.knaw.dans.easy.bag2deposit.BagSource._
-import nl.knaw.dans.easy.bag2deposit.Fixture.{ AppConfigSupport, FileSystemSupport }
+import nl.knaw.dans.easy.bag2deposit.Fixture.{AppConfigSupport, FileSystemSupport, XmlSupport}
 import nl.knaw.dans.easy.bag2deposit.IdType._
+import org.scalatest.OneInstancePerTest
 import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
 import scalaj.http.HttpResponse
 
-import scala.util.Success
+import scala.util.{Success, Try}
+import scala.xml.XML
 
-class AppSpec extends AnyFlatSpec with Matchers with AppConfigSupport with FileSystemSupport {
+class AppSpec extends AnyFlatSpec with XmlSupport with Matchers with AppConfigSupport with FileSystemSupport with OneInstancePerTest {
   private val resourceBags: File = File("src/test/resources/bags/01")
+  private val resourceBags_2: File = File("src/test/resources/bags/02")
   private val validUUID = "04e638eb-3af1-44fb-985d-36af12fccb2d"
 
-  "addPropsToBags" should "move valid exports" in {
-    val delegate = mock[MockBagIndex]
-    val noBaseBagUUID = "87151a3a-12ed-426a-94f2-97313c7ae1f2"
-    (delegate.execute(_: String)) expects s"bag-sequence?contains=$validUUID" returning
-      new HttpResponse[String]("123", 200, Map.empty)
-    (delegate.execute(_: String)) expects s"bag-sequence?contains=$noBaseBagUUID" returning
-      new HttpResponse[String]("123", 200, Map.empty)
-    (delegate.execute(_: String)) expects s"bags/4722d09d-e431-4899-904c-0733cd773034" returning
-      new HttpResponse[String]("<result><bag-info><urn>urn:nbn:nl:ui:13-z4-f8cm</urn><doi>10.5072/dans-2xg-umq8</doi></bag-info></result>", 200, Map.empty)
-    val appConfig = testConfig(delegatingBagIndex(delegate))
+  val delegate = mock[MockBagIndex]
+  val noBaseBagUUID = "87151a3a-12ed-426a-94f2-97313c7ae1f2"
+  (delegate.execute(_: String)) expects s"bag-sequence?contains=$validUUID" returning
+    new HttpResponse[String]("123", 200, Map.empty)
+  (delegate.execute(_: String)) expects s"bag-sequence?contains=$noBaseBagUUID" returning
+    new HttpResponse[String]("123", 200, Map.empty)
+  (delegate.execute(_: String)) expects s"bags/4722d09d-e431-4899-904c-0733cd773034" returning
+    new HttpResponse[String]("<result><bag-info><urn>urn:nbn:nl:ui:13-z4-f8cm</urn><doi>10.5072/dans-2xg-umq8</doi></bag-info></result>", 200, Map.empty)
+  val appConfig = testConfig(delegatingBagIndex(delegate), null)
```

Comment:
Move this to a method called by those tests that need it rather than OneInstancePerTest.

Reply:
done